pyqtgraph/debug.py

```diff
@@ -126,7 +126,30 @@ def printTrace(msg='', indent=4, prefix='|'):
 
 def backtrace(skip=0):
     return ''.join(traceback.format_stack()[:-(skip+1)])    
+
+
+def formatException(exctype, value, tb, skip=0):
```

Comment:
Can you detail what the purpose of `skip` is? The first three arguments are the positionals to traceback.format_exception and traceback.TracebackException IIUC